Scripts/Editor/BSoftBodyEditor.cs

```diff
@@ -222,5 +223,80 @@ void DrawCustomMeshSettingsOptions()
             EditorGUILayout.Space();
         }
 
+        private int _currentSelectedNode = -1;
+        
+        private void OnSceneGUI()
+        {
+            
+            SoftBody softBody = bSoftBodyTarget.GetCollisionObject() as SoftBody;
+            MeshFilter meshFilter = bSoftBodyTarget.GetComponent<MeshFilter>();
+//            Transform trans = bSoftBodyTarget.transform;
+            Vector3[] vertices = meshFilter.sharedMesh.vertices;
```

Comment:
This looks cool!

Can the vertices of the shared mesh be cached? Getting the vertices every editor frame update will generate a fair bit of garbage.

Reply:
That's definitely a good idea, but I am actually not using the vertices at all! I was thinking about using them, but I'm using the nodes instead. 

Do you have any suggestions on other properties that should be available in the editor view?

Reply:
Are there some stiffness properties that could be edited this way? That would make adjusting the properties very intuitive.